js/src/mesh.ts

```diff
@@ -54,13 +78,31 @@ class MeshView extends widgets.WidgetView {
                 animation_time_texture : { type: "f", value: 1. },
                 texture: { type: "t", value: null },
                 texture_previous: { type: "t", value: null },
-        };
+            },
+            THREE.UniformsLib[ "common" ],
+            THREE.UniformsLib[ "lights" ],
+            {
+                emissive: { value: new THREE.Color( 0x000000 ) },
+                emissiveIntensity: { value: 1 },
+                specular: { value: new THREE.Color( 0xffffff ) },
+                shininess: { value: 0 },
+                roughness: { value: 0.0 },
+                metalness: { value: 0.0 },
+            },
+            
+        ] );
+
         const get_material = (name)  => {
             if (this.model.get(name)) {
                 return this.model.get(name).obj.clone();
             } else {
                 const mat = new THREE.ShaderMaterial();
                 mat.side = THREE.DoubleSide;
+
+                mat.flatShading = false;
+                mat.transparent = true;
```

Comment:
Why do we need to set tranparant to True? Can't we use Mesh.material.transparent ?